app/scripts/services/acceptanceTest.js

```diff
@@ -102,7 +96,7 @@ angular.module('ludwig').factory('AcceptanceTestsService', function($q, $http, $
             return $http.get(config.baseApiPath + (isPublic ? '/public' : '') + '/acceptance-tests', { params: filters }).then(function(result) {
                 var tests = result.data;
                 _.map(tests, formatValues);
-                return orderTestsByKeywords(tests);
+                return tests;
```

Comment:
Don't hesitate to `return` a function call: here, the function body would probably be more readable as `return _map(result.data, formatValues);`  :)